lms/djangoapps/course_api/api.py

```diff
@@ -0,0 +1,58 @@
+"""
+Course API
+"""
+
+from django.contrib.auth.models import User
+from lms.djangoapps.courseware.courses import (
+    get_courses,
+    get_course_by_id
+)
+from opaque_keys.edx.keys import CourseKey
+
+from .serializers import CourseSerializer
+
+
+def _has_permission(requesting_user, username):
```

Comment:
Can we come up with a better name for this method?  Can it be specific to the permission that it's checking?